File: Self.Improvement.Data/Entities/Message.cs
using System;
using Self.Improvement.Data.Enums;

namespace Self.Improvement.Data.Entities
{
    public class Message : IEntity
    {
        public Guid Id { get; set; }
        public string Text { get; set; }
        public int TelegramChatId { get; set; }
        public bool FromBot { get; set; }
        public DateTime Date { get; set; }
        public Guid ChatId { get; set; }
        public MessageStatus Status { get; set; }
    }
}

File: Self.Improvement.Web/ClientApp/src/app/shared/types/message.ts
import MessageStatus from './enums/message-status.enum';

export default class Message {
    public id: string;
    public text: string;
    public chatId: string;
    public telegramChatId: number;
    public fromBot: boolean;
    public date: Date;
    public status: MessageStatus;
}
